src/Events/Services/Interfaces/IAppEventsService.cs

```diff
@@ -12,14 +12,30 @@ namespace Altinn.Platform.Events.Services.Interfaces
     public interface IAppEventsService
     {
         /// <summary>
-        /// Stores a cloud event document to the events database.
+        /// Save cloud event to persistent storage.
         /// </summary>
-        /// <param name="cloudEvent">The cloudEvent to be stored</param>
+        /// <param name="cloudEvent">The cloudEvent to be saved</param>
         /// <returns>Id for the created document</returns>
-        Task<string> StoreCloudEvent(CloudEvent cloudEvent);
+        Task<string> SaveToDatabase(CloudEvent cloudEvent);
+
+        /// <summary>
+        /// Push cloud event to inbound queue.
+        /// Should only be attempted after saving to persistent storage. 
+        /// </summary>
+        /// <param name="cloudEvent">The cloudEvent to be queued</param>
+        /// <returns>Id for the queued event</returns>
+        Task<string> PushToInboundQueue(CloudEvent cloudEvent);
```

Comment:
Pushing to inbound queue is agnostic w.r.t. event type, so maybe this would be the time to create a new IEventService which handles push to InboundQueue that can be reused for external events as well ?